src/Engine.php

```diff
@@ -155,12 +155,15 @@ private function runMutationAnalysis(): void
                 : []
         );
 
-        $actualExtraOptions = $this->config->getTestFrameworkExtraOptions();
+        $extraOptions = $this->config->getTestFrameworkExtraOptions();
 
-        $filteredExtraOptionsForMutant = $this->adapter instanceof ProvidesInitialRunOnlyOptions
-            ? $this->testFrameworkExtraOptionsFilter->filterForMutantProcess($actualExtraOptions, $this->adapter->getInitialRunOnlyOptions())
-            : $actualExtraOptions;
+        $mutationExtraOptions = $this->adapter instanceof ProvidesInitialRunOnlyOptions
+            ? $this->testFrameworkExtraOptionsFilter->filterForMutationProcess(
+                $extraOptions,
+                $this->adapter->getInitialRunOnlyOptions()
+            )
+            : $extraOptions;
 
-        $this->mutationTestingRunner->run($mutations, $filteredExtraOptionsForMutant);
+        $this->mutationTestingRunner->run($mutations, $mutationExtraOptions);
```

Comment:
The real change here is `filterForMutantProcess` -> `filterForMutationProcess`, the rest is pure CS